ext/symengine/symengine.c

```diff
@@ -44,6 +45,9 @@ void Init_symengine() {
     rb_define_method(c_basic, "subs", cbasic_subs, -1);
     rb_define_method(c_basic, "coerce", cbasic_coerce, 1);
 
+    //Sympify as a Module Level Function
+    rb_define_module_function(m_symengine, "sympify", cutils_sympify, 1);
```

Comment:
@isuruf yeah, I've meant the former, and it can be done just by defining method `SymEngine` (in the main namespace, not under SymEngine module). Ruby core classes do it that way, for ex. `Array()` or `Integer()` (and their exactly mean is "convert to this type", which in our case is close to what `sympify` does).